packages/sirius-web/frontend/sirius-web/src/index.tsx

```diff
@@ -13,6 +13,17 @@
 import { loadDevMessages, loadErrorMessages } from '@apollo/client/dev';
 import { ExtensionRegistry } from '@eclipse-sirius/sirius-components-core';
 import { NodeTypeContribution } from '@eclipse-sirius/sirius-components-diagrams';
+import {
```

Comment:
The widget reference is not optional for all those who depend on `sirius-web-application` (think SysON, Papyrus Web) to its frontend support must not be optional either, as such all the changes in this file should be made in the DefaultExtensionRegistry from `sirius-web-application`.

Now that I'm thinking about it, since the Ellipse node is not optional either since its a dependency of `sirius-web-application` on the backend, it's frontend support should also be in the default extension registry.

On both the frontend and the backend, the `sirius-web` projects are for experiments for example with Papaya, Flow, Task etc. Core features should be in `sirius-web-application`.

Reply:
The custom nodes aren't contribute with an extension point for now, so I have to leave it in sirius-web.
I propose to add this extension point in a new PR